packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewer/typings.d.ts

```diff
@@ -15,20 +15,21 @@ declare module 'pdfjs-dist/lib/web/ui_utils' {
 
 declare module 'pdfjs-dist/lib/web/text_layer_builder' {
   import { EventBus } from 'pdfjs-dist/lib/web/ui_utils';
-  import PdfjsLib from 'pdfjs-dist';
+  import { PageViewport } from 'pdfjs-dist/types/display/display_utils';
```

Comment:
Is this file still needed? Or can we just remove this and get what we need from upgraded library's typings?